src/scenes/NurseryRouter/NurseryWithdrawalsTable.tsx

```diff
@@ -141,12 +142,40 @@ export default function NurseryWithdrawalsTable(): JSX.Element {
     void getApiSearchResults();
   }, [selectedOrganization]);
 
+  const getWithdrawalBy = useCallback(
+    (withddrawalId: unknown) => {
+      const found = searchResults?.find((withdrawal) => withdrawal.id === withddrawalId);
+      return found;
+    },
+    [searchResults]
+  );
+
+  useEffect(() => {
+    const withdrawalsWithUndoDate: SearchResponseElement[] = [];
+    searchResults?.reduce((acc, withdrawal) => {
+      if (withdrawal.undoesWithdrawalId) {
+        acc.push({
+          ...withdrawal,
+          undoesWithdrawalDate: getWithdrawalBy(withdrawal?.undoesWithdrawalId)?.withdrawnDate,
```

Comment:
I missed that we needed to show this date. I can add it as a search field so we can avoid the extra round-trip here.

Reply:
https://github.com/terraware/terraware-server/pull/1856 adds `undoesWithdrawalDate` and `undoneByWithdrawalDate` search fields.

Reply:
Thanks steve! Will update this